pkgs/by-name/qs/qsv/package.nix

```diff
@@ -0,0 +1,138 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, sqlite
+, zstd
+, stdenv
+, darwin
+, python3
+, file
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "qsv";
+  version = "0.128.0";
+
+  src = fetchFromGitHub {
+    owner = "jqnatividad";
+    repo = "qsv";
+    rev = version;
+    hash = "sha256-DoCiBERnFWCxJuTLJV3gWMzaukpgqI82L9GTnF/UZyI=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "calamine-0.25.0" = "sha256-31fJ6wS0s9lyHqf/D4Po6upkOk2IIG3gXjV/FropwlI=";
+      "dynfmt-0.1.5" = "sha256-/SrNOOQJW3XFZOPL/mzdOBppVCaJNNyGBuJumQGx6sA=";
+      "grex-1.4.5" = "sha256-cGO40UFjm7/mjOqxBPZG5N8h9/BmQXSu5aHnNrezfXE=";
+      "polars-0.40.0" = "sha256-znAavcdg4OM5wa0OpagEysSZCAZcuhMZ1RpLghkm9nU=";
+    };
+  };
+
+  buildInputs =
+    [
+      file
+      sqlite
+      zstd
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      darwin.apple_sdk.frameworks.CoreFoundation
+      darwin.apple_sdk.frameworks.IOKit
+      darwin.apple_sdk.frameworks.Security
+      darwin.apple_sdk.frameworks.SystemConfiguration
+    ];
+
+  nativeBuildInputs = [
+    pkg-config
+    python3
+    rustPlatform.bindgenHook
+  ];
+
+  buildNoDefaultFeatures = true;
+
+  buildFeatures = if !stdenv.isDarwin then [ "all_features" ] else [
+    "apply"
+    "feature_capable"
+    "fetch"
+    "foreach"
+    "geocode"
+    "polars"
+    "python"
+    "self_update"
+    "to"
+  ];
```

Comment:
What's different about darwin here? Also there is a whole discussion to be had about what features are included by default... reading through https://github.com/jqnatividad/qsv/blob/master/docs/FEATURES.md, I'd take out `polars`, `python`, `self_update` (doesn't make sense if installed with Nix right?) from what you have here.

Reply:
Some feature prevented a successful build on Darwin. Not entirely sure why. My goal there was to get something that builds and mostly works but I haven't tested on that platform since the initial PR. I'm fine with simplifying this to a single set of features and fixing the compatibility issue later.

Removing polars, python, and self_update all make sense to me.